.gitlab-ci.yml

```diff
@@ -15,8 +15,9 @@ variables:
 default:
   before_script:
     - apk add --no-cache --upgrade git bash findutils grep curl jq shfmt rsync diffutils gawk
+    - git config --global --add safe.directory "*" # fixme?
     - git remote set-url origin $REPO_URL
-    - git fetch --tags --all --depth 20
+    - git fetch --tags --force --all --depth 20 # fixme?
```

Comment:
This probably also isn't necessary, it only broke because you had before_script both in global and the individual local namespace

Reply:
This refers to a clobbered tag? It couldn't fetch the tag due to this error, I was assuming this to be due to force pushing at the time.